evalai/utils/submissions.py

```diff
@@ -18,6 +21,91 @@
 requests.packages.urllib3.disable_warnings()
 
 
+def upload_submission_file_with_presigned_url(challenge_phase_pk, file, submission_metadata={}):
+    """
+    Function to upload a file to AWS using a presigned url
+    """
+    url = "{}{}".format(get_host_url(), URLS.get_submission_file_presigned_url.value)
+    url = url.format(challenge_phase_pk)
+
+    headers = get_request_header()
+
+    try:
+        data = {"status": "submitting", "file_name": file}
+        data = dict(data, **submission_metadata)
+        response = requests.post(
+            url, headers=headers, data=data
+        )
+        if response.status_code is not HTTPStatus.CREATED:
+            response.raise_for_status()
+
+        response = response.json()
+        presigned_url = response.get("presigned_url")
+        submission_pk = response.get("submission_pk")
+
+        # Uploading the submisison file to S3.
+        echo(
+            style(
+                "Your file is being uploaded...",
+                fg="green",
+                bold=False,
+            )
+        )
+        with open(os.path.realpath(file), 'rb') as f:
+            try:
+                response = requests.put(
+                    presigned_url,
+                    data=f
+                )
+                response.raise_for_status()
+            except requests.exceptions.HTTPError as err:
+                if response.status_code is not HTTPStatus.OK:
+                    echo("There was an error while uploading the file: {}".format(err))
+                    sys.exit(1)
+
+        # Publishing the submisison message, for processing by the submission worker.
```

Comment:
change this to `Publishing submission message to the message queue for processing` ?